servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -842,6 +887,21 @@ public Builder retryOther(
             return this;
         }
 
+        /**
+         * Callback invoked on every {@link StreamingHttpClient#request(StreamingHttpRequest) request} retry attempt.
+         * <p>
+         * This can be used to track when {@link BackOffPolicy} actually decides to retry a request, to update
+         * {@link HttpRequestMetaData request meta-data} before a retry, or implement logging/metrics.
```

Comment:
consider also clarifying here that the implementor can't influence the retry decision at this point and needs to use other methods to do that.